examples/chart/teleport-cluster/templates/proxy/predeploy_job.yaml

```diff
@@ -17,6 +17,12 @@ metadata:
 spec:
   backoffLimit: 1
   template:
+    metadata:
+      labels:
+        {{- include "teleport-cluster.proxy.labels" . | nindent 8 }}
+        {{- if $proxy.extraLabels.jobPod }}
+        {{- toYaml $proxy.extraLabels.jobPod | nindent 8 }}
+        {{- end }}
```

Comment:
```suggestion
        {{- with $proxy.extraLabels.jobPod }}
        {{- toYaml . | nindent 8 }}
        {{- end }}
```

Reply:
why with instead of if (seems the chart is using "if" such as line 9 in the same file

Reply:
as discuss let's keep if instead of with to keep stuff already did in the chart